setup.py

```diff
@@ -60,12 +53,15 @@ def main():
           extras_require={
               'python-daemon': ['python-daemon'],
           },
-          packages=find_packages(exclude=['bin']),
+          packages=find_packages(exclude=['bin', 'test']),
           scripts=['bin/ssmreceive', 'bin/ssmsend'],
           data_files=[(conf_dir, conf_files),
                       ('/etc/logrotate.d', ['conf/apel-ssm']),
-                      ('/etc/init.d', ['bin/apel-ssm']),
-                      ('/usr/share/doc', ['apel-ssm'])],
+                      ('/usr/share/doc/apel-ssm', ['apel-ssm']),
```

Comment:
Where'd the docs go before this change?

Reply:
it went in a file under `/usr/share/doc` rather than a file under `/usr/share/doc/apel-ssm`.

Reply:
Ok, that sounds better now.